tests/interfaces/records/handlers/records-query.spec.ts

```diff
@@ -84,6 +85,37 @@ describe('handleRecordsQuery()', () => {
       expect(reply2.entries?.length).to.equal(1); // only 1 entry should match the query
     });
 
+    it('should be able to query by attester', async () => {
```

Comment:
Consider testing a couple additional behaviors: 
* When there are no matches to the filter.
* A filter with additional properties, so it's clear that it's and AND filter.

Reply:
Done.